mood-analysis.rb

```diff
@@ -19,6 +19,52 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(sentence)
+  sentence.gsub(/[[:punct:]]/, "")
+end
+
+def happy_days(journal)
+  entries = 0
+  happy_count = 0
+  # wanted to do this with reduce but didn't get it figuerd out :/
+  journal.each do |entry|
+    happy_count += 1 if analyze_mood(strip_punctuation(entry)) == ":-)"
+    entries += 1
+    if happy_count > 3
+      break
+    end
+  end
+  if happy_count < 3
+    puts "In all #{entries} enterest, there were not 3 happy days! Sorry :-(."
+  else
+    puts "It takes #{entries} entries for 3 happy days to occur."
+  end
+end
```

Comment:
I would rather this method only return a fixnum and not a string with the number buried inside. Makes the information able to be used in more ways than this specific case.